### PR TITLE
Add score (resolves #3)

### LINKED ISSUE #3: Display current score
The score is the lenght of the snake. It should be displayed on the screen

### PR DESCRIPTION
**Pull request by jlesauce**
I have the pleasure to announce that the feature "Score" is now finished by our teams!

Closes #3

### CODE REVIEW

**Comment by jlesauce** [comment]
Hello! I like pizza :)

File: index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <body onload="start()" style="background-color: black">  
         <a href="https://github.com/ourtigarage/web-snake"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/652c5b9acfaddf3a9c326fa6bde407b87f7be0f4/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6f72616e67655f6666373630302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_orange_ff7600.png"></a>            
         <center><canvas id="game" width="550" height="550" style="background-color: white; height: 90%"></canvas></center>
+        <left><font color="yellow">Score: <span id="score">12</span></font></left>
```
Comment (phsym) [inline index.html]: Initialize to 0

File: js/snake.js [hunk 2]
```diff
@@ -90,6 +91,8 @@ class Game {
         //Check if next move is on some food
         if(newHead[0] == this.food[0] && newHead[1] == this.food[1]) {
             console.log("Yum yum !!!");
+            this.score.increment();
```
Comment (phsym) [inline js/snake.js]: What about using 
```javascript
this.snake.lenght()
```
to compute the score ?

File: js/snake.js [hunk 3]
```diff
@@ -140,3 +146,22 @@ function loop(game, ctx) {
     game.update();
     game.render(ctx);
 }
+
+class Score {
+    constructor() {
```
Comment (phsym) [inline js/snake.js]: I would probably pass the html element `id` as constructor argument here

**Review by phsym — CHANGES_REQUESTED** [review]

**Comment by phsym** [comment]
Also, please resolv conflict :pizza:

File: index.html (hunk 1 above)
Comment (jlesauce) [reply index.html]: done

File: js/snake.js (hunk 2 above)
Comment (jlesauce) [reply js/snake.js]: done

File: js/snake.js (hunk 3 above)
Comment (jlesauce) [reply js/snake.js]: Score class removed

File: index.html [hunk 4]
```diff
@@ -8,13 +8,14 @@
 	    	<div style="position: absolute; top: 300; border: 0; background-color: rgba(88,88,88,0.4);padding: 25px;">
 	    		<center>
 	    			<H1>Game in pause</H1>
-				<H1>Have Fun at the Hacktoberfest!</H1>
+            <H1>Have Fun at the Hacktoberfest!</H1>
```
Comment (phsym) [inline index.html:11]: indent

**Review by phsym — APPROVED** [review]

**Pull request merged**